### ISSUE TITLE
Current and EMF are being used interchangeably

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/faradays-electromagnetic-lab#118]
As part of #103, I noticed that current and EMF are being used without any mention of resistance or Ohm's law. There is an important nuance that is not mentioned in the documentation:

- The flux and change in flux are proportional to the number of loops in the pickup coil (N)
- The EMF induced in the pickup coil is proportional to the change in flux, so it is also proportional to N.
- The resistance of the pickup coil is proportional to the total wire length, so it too is proportional to N.
- The current in the pickup coil is equal to the EMF divided by the resistance, so the factors of N cancel out. This makes the current in the pickup coil independent of the number of loops.

These relationships, and the pivotal role of resistance, are not mentioned in the model documentation. Additionally, it seems that Ohm's law is being glossed over with code such as the following:

```

this.currentAmplitudeProperty = new DerivedProperty( [ this.emfProperty, this.maxEMFProperty ],
      ( emf, maxEMF ) => {
        const currentAmplitude = emf / maxEMF;
        return FELConstants.CURRENT_AMPLITUDE_RANGE.constrainValue( currentAmplitude );
      }

```

I recommend updating the model documentation, and likely the names of a few variables, to make the physics calculations more clear and fully explained.

**Comment by pixelzoom** [comment]
Current and EMF are not being used interchangeably.  There is no current in the model.   There is currentAmplitudeProperty, which is something entirely different, as documented.

Ohm's Law is not only "glossed over", it is intentionally omitted.  As noted in https://github.com/phetsims/faradays-electromagnetic-lab/issues/117#issuecomment-2020950981, model.md explicitly says that we are not modeling resistance:

> ### Electromagnet
> ... Note that there is no model of resistance for the coil or voltage source.

We are also not modeling current in either coil.  Our Hollywood model is based on something that we call "current amplitude",  documented in model.md as:

> **Current amplitude**: A percentage with range [-1,1] that describes the amount of current relative to the 
maximum current that may be induced in the model. The sign indicates the direction 
of the current. View components use this value to determine how they should respond to induced current. 
For example, deflection of the voltmeter needle, brightness of the light bulb, and speed of
electrons

... and in Coli.ts:

>   // This is a quantity that PhET made up. It is a percentage [-1,1] that describes the amount of current relative to
  // some maximum current in the model, and the sign indicates the direction of that current. View components can use
  // this value to determine how they should behave -- eg, how far to move a voltmeter needle, how bright to make a
  // light bulb, and how fast to move electrons.
  public readonly currentAmplitudeProperty: TReadOnlyProperty<number>;
  private readonly currentAmplitudeRange: Range;

We have also refrained from providing units for `fluxProperty`, `deltaFluxPropert` and `emfProperty`, because their values are not at all realistic -- they are a Hollywooded version of Faraday's Law.  

@arouinfar How do you want to proceed?

**Comment by pixelzoom** [comment]
Discussed with @arouinfar. Adding resistance and current to the model is not something we want to do. More at https://github.com/phetsims/faradays-electromagnetic-lab/issues/119#issuecomment-2021675396.

@matthew-blackman @samreid What do you think we should do next for this issue?

**Comment by samreid** [comment]
The proposed change in #130 will address part of this issue.

**Comment by pixelzoom** [comment]
Discussed with @matthew-blackman and @samreid.  The resolution for this issue is to improve the documentation in model.md.  @matthew-blackman will take a stab at that in https://github.com/phetsims/faradays-electromagnetic-lab/issues/70, and @arouinfar and I will review.

@matthew-blackman Anything else here? If not, feel free to close.

**Comment by matthew-blackman** [comment]
Nothing else from me. This is being addressed in #70. Closing.
